config/routes.rb

```diff
@@ -0,0 +1,16 @@
+Rails.application.routes.draw do
+  get 'splash/index'
+  get 'foods/index'
+  devise_for :users, controllers: {
+    sessions: 'users/sessions',
+    registrations: 'users/registrations'
+  }
+  resources :users
+  resources :splash
```

Comment:
- [ ] Please remove all the routes that you are not using from the `config/routes.rb` file to enhance code clarity, security, and performance. I suggest that you remove the `resources :users` and `resources :splash` since we are not using them.

Reply:
Thanks @LionRouge1, the changes asked by you has been done. 😃